From 92d0babf0a4548b0d12553ab7005e66a19cb7c45 Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Tue, 16 Jun 2009 13:02:47 +0800
Subject: [PATCH] ENGR00113366 MX27ads: Fix compiling error

This error is caused by ENGR00093265
error: conflicting types for 'mxc_dma_request_ext' between
dma_mx2.c and dma.h

Signed-off-by: Wallace Wang <r59996@freescale.com>
---
 arch/arm/plat-mxc/dma_mx2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/dma_mx2.c b/arch/arm/plat-mxc/dma_mx2.c
index bca527b..5a7e93b 100644
--- a/arch/arm/plat-mxc/dma_mx2.c
+++ b/arch/arm/plat-mxc/dma_mx2.c
@@ -772,7 +772,7 @@ static inline int consume_dma_bd(mxc_dma_channel_t * dma, int error)
  *         succeed, returns the channel number to be used on success.
  */
 int mxc_dma_request_ext(mxc_dma_device_t channel_id, char *dev_name,
-			unsigned long data)
+			struct dma_channel_info *info)
 {
 	mxc_dma_channel_t *dma;
 	mx2_dma_priv_t *dma_private = NULL;
-- 
1.5.4.4

